rpm/cray/csm/noos/index.yaml

```diff
@@ -44,8 +44,8 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
     - csm-node-identity-1.0.22-1.noarch
     - csm-ssh-keys-1.6.3-1.noarch
     - csm-ssh-keys-roles-1.6.3-1.noarch
-    - csm-testing-1.18.9-1.noarch
-    - goss-servers-1.18.9-1.noarch
+    - csm-testing-1.18.*-1.noarch
+    - goss-servers-1.18.*-1.noarch
```

Comment:
Temporary change to test glob processing. Represents a sample change which we may have in integration branch. Will be reverted before merge into release branch.